bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
This is catching any type of Exception and calling it AuthError (a context scoped variable name, not a type), which is not what we want to do here. With this code, any Exception raised will tell the user that their password is wrong. For example, an ```IOError``` would print "wrong password, please try again". We need to catch only the Exception that is raised when the password is wrong and then print such a message.